chemie/quiz/models.py

```diff
@@ -33,4 +33,4 @@ class QuizScore(models.Model):
     score = models.IntegerField(default=0)
 
     def __str__(self):
-        return str(self.score)
+        return str(self.term) + ", " + str(self.user) + ", " + str(self.score)
```

Comment:
This should be changed back. If you look at how the Quiz score is displayed in the template, you will see that this becomes weird.